### PR TITLE
 Multiple instances in the same page.

### PR DESCRIPTION
**Pull request by ChathuraGH**
I am new to pull requests😏. If you can, please guide me little.👌. I created a pull request with my new zuck.js,zuck.css,guide.txt files.💖.

### CODE REVIEW

File: dist/zuck.css [hunk 1]
```diff
@@ -1,651 +1,109 @@
...
-  top: 0;
-  left: 0;
-  width: 100vw;
-  height: 100%;
-  background: rgba(0, 0, 0, 0.75);
-  z-index: 100000;
-  font-size: 14px;
-  font-family: inherit;
-}
-#zuck-modal-content,
```
Comment (ChathuraGH) [inline dist/zuck.css:35]: Id selection changed to class selection.

**Comment by ChathuraGH** [comment]
What is going on friend? 😔😊.

**Comment by ramonszo** [comment]
Hey @ChathuraGH I can't merge this PR because looks like you're editing the `dist/` compiled file instead of `src/` file. 

Here's a example PR that is doing correctly: https://github.com/ramonszo/zuck.js/pull/147

**Comment by ChathuraGH** [comment]
Thanks. I'm new to PR. That's why. 👍.

**Comment by ramonszo** [comment]
no worries @ChathuraGH. i appreciate your engagement with this repo

**Pull request closed without merging**